src/js/pageNotFound.jsx

```diff
@@ -52,15 +51,18 @@ function PageNotFound() {
 
 export function pageInit(args) {
   ReactDOM.render(
-    <PageLayout
-      role={args.role}
-      userLang={args.userLang}
-      username={args.username}
-      version={args.versionDeployed}
-      showSearch={false}
-    >
-      <PageNotFound />
-    </PageLayout>,
+    <Suspense fallback="">
+      <PageLayout
+        role={args.role}
+        username={args.username}
+        mapboxToken={args.mapboxToken}
+        mapquestKey={args.mapquestKey}
+        versionDeployed={args.versionDeployed}
+        showSearch={true}
```

Comment:
This should be `showSearch={false}`. I also don't think we need to provide the `mapboxToken` or `mapquestKey` here